extensions/wikia/UserLogin/FacebookSignupController.class.php

```diff
@@ -240,23 +253,6 @@ public function login() {
 	}
 
 	/**
-	 * Return Facebook account data like email, gender, real name
-	 */
-	public function getFacebookData() {
-		$fbUserId = $this->request->getVal( 'fbUserId' );
-
-		if ( $fbUserId > 0 ) {
-
-			$data = FacebookClient::getInstance()->getUserInfoAsArray( $fbUserId );
```

Comment:
If `getFacebookData` has been removed I think we can remove `getUserInfoAsArray` as well!